index.html

```diff
@@ -1,95 +1,84 @@
-	<meta http-equiv="cleartype" content="on">
-
-	<!-- Facebook meta tags-->
-	<meta property="og:title" content="350 Mega Map"/>
-	<meta property="og:type" content="website"/>
-	<meta property="og:url" content="http://map.350.org/"/>
-	<meta property="og:image" content="http://350.org/sites/all/themes/threefifty/logo.gif"/>
-	<meta property="og:site_name" content="350 Mega Map"/>
-	<meta property="og:description" content="Mapping a variety of local 350 groups, events and climate actions around the world." />
-
-	<!-- Twitter meta tags -->
-	<meta property="twitter:site" content="@350">
-	<meta property="twitter:card" content="350 Mega Map">
-	<meta property="twitter:url" content="http://map.350.org">
-	<meta property="twitter:title" content="350 Mega Map">
-	<meta property="twitter:description" content="Mapping a variety of local 350 groups, events and climate actions around the world.">
-	<meta property="twitter:image" content="http://350.org/sites/all/themes/threefifty/logo.gif">
-
-        <link rel="stylesheet" href="css/leaflet.css" />
-        <!--[if lte IE 8]>
-            <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.ie.css" />
-            <![endif]-->
-            <!--[if IE]>
-            <style type="text/css">.leaflet-top { top: 60px; }</style>
-            <![endit]-->
-        <script src="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"></script>
-
-	<!-- Javascript -->
-	<script type="text/javascript" src="js/jquery-1.9.1.min.js"></script>
-
-        <script type="text/javascript" src="markercluster/leaflet.markercluster.js"></script>
-        <link rel="stylesheet" href="markercluster/MarkerCluster.css" />
-        <link rel="stylesheet" href="markercluster/MarkerCluster.Default.css" />
-        <!--[if lte IE 8]><link rel="stylesheet" href="markercluster/MarkerCluster.Default.ie.css" /><![endif]-->
-
-        <script src="//cdn.jsdelivr.net/leaflet.esri/1.0.3/esri-leaflet.js"></script>
-        
-
-        <script src="geosearch/js/l.control.geosearch.js"></script>
-        <script src="geosearch/js/l.geosearch.provider.google.js"></script>
-        <link rel="stylesheet" href="geosearch/css/l.geosearch.css" />
-
-        <script src="addmarker/Control.AddMarker.js"></script>
-        <link rel="stylesheet" href="addmarker/addmarker.css" />
-        <!--[if lte IE 8]><link rel="stylesheet" href="addmarker/addmarker.ie.css" /><![endif]-->
-
-        <script src="js/leaflet.control.legendlayers.js"></script>
-
-	<link rel="stylesheet" href="css/bootstrap.min.css" />
-	<link rel="stylesheet" href="css/bootstrap-responsive.min.css" />
-	<link rel="stylesheet" href="//s3.amazonaws.com/s3.350.org/images/350-megamap-style.css" />
-
-	<script type="text/javascript" src="js/tabletop.js"></script>
-	<script type="text/javascript" src="js/handlebars.js"></script>
-
-        <script id="handlebars_template" type="text/x-handlebars-template">

```

Comment:
I think the question we should ask is if there is a need to have different tooltips based on the page. Or is it a different tooltip per type of event / map point. If it's the second I think moving it into the JS might be more useful and then we can add logic to make it switch tooltips appropriately.